fastapi/app/main.py

```diff
@@ -28,6 +28,13 @@ class ResponseModel(BaseModel):
     statusCode: int
     body: ResultModel | ErrorModel
 
+# Helper funcs
+def validate_quality(quality: str) -> str:
+    '''Check for empty strings, return default "smithed"'''
+    if (quality):
+        return quality
+    return "smithed"
```

Comment:
if we are going to validate, should we not also check if it's a valid type not just empty? Also, this can be added to the C++ under utils.cpp and it'll be automatically wrapped in python. We could call it within the the search call where the qualities are being used not have it be called everywhere.

Reply:
It seems like there's already a check for valid qualities in spice_types.cpp and the [exception response](https://github.com/DOI-USGS/SpiceQL/blob/main/SpiceQL/src/spice_types.cpp#L84) is the one I received when an empty string was passed. I can still move this validation check from main.py to the Kernel::translateQuality() func unless we want it in utils.cpp. Should a non-empty, non-valid quality default to "smithed" instead of throwing an exception listing valid options?

Reply:
It would depend on why we are getting empty strings in the first place, theoretically we should treat it as a non-parameter and therefore default to smithed.